secator/query/sqlite.py

```
@@ -134,6 +134,20 @@ def _execute_search(self, query: dict, limit: int = 0, exclude_fields: list = No
 			console.print(Warning(message=f'SQLite search failed: {e}'))
 			return []
 
+	def _execute_iterate(self, query: dict, batch_size: int = 1000):
+		"""Stream rows with a cursor + fetchmany — never loads all N into memory."""
+		try:
+			conn = self._get_conn()
+			where, params = _build_where(query)
+			cur = conn.execute(f"SELECT data FROM findings WHERE {where or '1=1'}", params)
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)